src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,38 @@
      * Андрей - 600
      * София - 900</p>
      */
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException {
+        final DateTimeFormatter datePattern = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
make it constant

Reply:
https://stackoverflow.com/questions/66066/what-is-the-best-way-to-implement-constants-in-java